### DISCUSSION TITLE
Zmiana metody /sale/offer-quantity-change-commands

### DISCUSSION THREAD

**Discussion by keczerad** [allegro/allegro-api#8016] [Suggestions - resolved]
Obecnie jest tak:

![image](https://github.com/allegro/allegro-api/assets/22832921/216e0c54-5ef5-4803-ade8-986253975d19)

Moim zdniem powinno być tak:

![image](https://github.com/allegro/allegro-api/assets/22832921/95d892d3-5764-4e62-b647-27f6e592242b)

Wtedy ta metoda miała by sens.

**Comment by Aureliusz-Brussy** [comment]
Wdrożyliśmy taką funkcjonalność, [więcej przeczytasz w poradniku](https://developer.allegro.pl/tutorials/jak-zarzadzac-ofertami-7GzB2L37ase#cena-i-liczba-przedmiotow).
